openpype/hosts/maya/plugins/publish/collect_arnold_scene_source.py

```diff
@@ -10,21 +10,23 @@ class CollectArnoldSceneSource(pyblish.api.InstancePlugin):
     # Offset to be after renderable camera collection.
     order = pyblish.api.CollectorOrder + 0.2
     label = "Collect Arnold Scene Source"
-    families = ["ass"]
+    families = ["ass", "assProxy"]
 
     def process(self, instance):
-        objsets = instance.data["setMembers"]
+        instance.data["members"] = []
```

Comment:
Isn't the `members` you are collecting here essentially what `instance[:]` is? (Although that might include parent hierarchy)

Reply:
Yeah, it includes the parent hierarchy, which we do not want. Got a suggestion?

Reply:
If a creator attribute `includeParentHierarchy` is `False` then `instance[:]` should not include the parent hierarchy, see [here](https://github.com/BigRoy/ayon-core/blob/f0ba7e62723a2fb0e502ae59d4ad6d52f7007b1b/client/ayon_core/hosts/maya/plugins/publish/collect_instances.py#L58-L59).

Whether that's relevant for this use case I'm not sure.

I was just commenting on these lines since it felt like duplicated logic - and in a way it is, but maybe there's a reason for it to be.

Reply:
Tried with `includeParentHierachy` but `instance[:]` also includes the proxy set, so this code is still needed.